src/pages/api/blocks/[...heights].ts

```diff
@@ -0,0 +1,26 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import { IndexerQuerier } from '../../../utils/indexer/connector';
+
+const indexerEndpoint = process.env['PENUMBRA_INDEXER_ENDPOINT'] ?? '';
+if (!indexerEndpoint) {
+  throw new Error('PENUMBRA_INDEXER_ENDPOINT is not set');
+}
+
+interface QueryParams {
+  heights?: string[];
+}
+
+export default async function blockInfoFetchHandler(req: NextApiRequest, res: NextApiResponse) {
```

Comment:
suggestion: move this into the `app` directory while applying the newer NextJS's API.

I believe we should discontinue supporting the `pages` directory and move everything related to v2 to the `app`. It's not just the new API but also, if done correctly, a performance increase.